### ISSUE TITLE
November 9 Firefox Reality Demo Issues

### ISSUE THREAD

**Issue by fwei-io** [hms-ecosystem/OpenXR-SDK#22]
@MortimerGoro please review, thanks.

![Image03](https://user-images.githubusercontent.com/57159719/141384946-c8dabb4f-c5d3-4f27-97d1-08e4d9ae1096.jpg)
![Image 2021-11-11 15-10-32](https://user-images.githubusercontent.com/57159719/141384949-a95fb796-3122-49d1-a65d-59d55d868a55.jpg)
![Image 2021-11-11 15-36-11](https://user-images.githubusercontent.com/57159719/141384951-050d6675-8494-4307-ad1f-2f193454f5c9.jpg)
![Image 2021-11-11 15-17-28](https://user-images.githubusercontent.com/57159719/141384953-faf50361-0a66-49a8-be0e-71a74b482090.jpg)
![Image06](https://user-images.githubusercontent.com/57159719/141387023-53b3e407-86b5-4bae-862e-f725c00cade4.jpg)

**Comment by MortimerGoro** [comment]
Thanks for the bug reports, I'm looking into those issues. I'll comment the results here.

**Comment by MortimerGoro** [comment]
Report for each issue:
- Not working video: it's a Gecko bug, this is the issue to track it: https://bugzilla.mozilla.org/show_bug.cgi?id=1736793. I added some comments
- Not working buttons: Fixed in the latest demo. Note that I changed the mapping to only exit WebXR using the 'y' button from the left controller, similar to Oculus Quest. Let me know if that's ok with you
- framevr.io: Fixed
- Overlapping controllers: Fixed
- Tooltips: Fixed
